ua-parser.js

```diff
@@ -74,6 +74,15 @@ const parseUA = (userAgent, browsers) => {
   const versions = Object.keys(browsers[data.browser.id].releases);
   versions.sort(compareVersions);
 
+  // Certain Safari versions are backports of newer versions, but contain less
+  // features, particularly ones involving OS integration. We will ignore these
```

Comment:
It's not ignored, something is returned. I guess `inBcd` being false is the important thing?

Reply:
Can you update the comment to reflect this?